boilerplate.py

```diff
@@ -288,10 +288,14 @@ def format_value(value):
         'hot',
         'hsv',
         'jet',
+        'neoncity',
         'pink',
         'prism',
         'spring',
         'summer',
+        'sunset',
```

Comment:
The advantage of "sunrise" over "sunset" is that the map goes from dark to light; but maybe one of the other naming schemes will override all this anyway.  (Also, "sunrise" has a more upbeat, optimistic feel...)